Makefile

```diff
@@ -121,42 +88,6 @@ push-tag: # example usage: make push-tag remote=origin tag=v1.100.1
 	 	git push $(remote) "$${dir:2}/$(tag)"); \
 	done
 
-DEPENDABOT_PATH=./.github/dependabot.yml
-.PHONY: gendependabot
-gendependabot: # generate dependabot.yml
-	@echo "Recreate dependabot.yml file"
-	@printf "# File generated by \"make gendependabot\"; DO NOT EDIT.\n\n" > ${DEPENDABOT_PATH}
-	@printf "version: 2\n" >> ${DEPENDABOT_PATH}
-	@printf "updates:\n" >> ${DEPENDABOT_PATH}
-	@printf "  - package-ecosystem: \"github-actions\"\n    directory: \"/\"\n    schedule:\n      interval: \"daily\"\n" >> ${DEPENDABOT_PATH}
-	@echo "Add entry for \"/\""
-	@printf "  - package-ecosystem: \"gomod\"\n    directory: \"/\"\n    schedule:\n      interval: \"daily\"\n" >> ${DEPENDABOT_PATH}
-	@set -e; for dir in $(SUBMODULES); do \
-		(echo "Add entry for \"$${dir:1}\"" && \
-		  printf "  - package-ecosystem: \"gomod\"\n    directory: \"$${dir:1}\"\n    schedule:\n      interval: \"daily\"\n" >> ${DEPENDABOT_PATH} ); \
-	done
-
-.PHONY: license-check
-license-check: # check if license is applied to relevant files
-	$Q licRes=$$(for f in $$(find . -type f \( -iname '*.go' -o -iname '*.sh' -o -iname '*.yml' \)) ; do \
-	           awk '/Copyright Splunk Inc.|generated|GENERATED/ && NR<=3 { found=1; next } END { if (!found) print FILENAME }' $$f; \
-	   done); \
-	   if [ -n "$${licRes}" ]; then \
-	           echo "license header checking failed:"; echo "$${licRes}"; \
-	           exit 1; \
-	   fi
-
-.PHONY: mod-tidy
-mod-tidy: # go mod tidy for all modules
-	${call for-all-modules,$(GO) mod tidy}
-	@echo "$(GO) mod tidy in $(TOOLS_MODULE_DIR)"
-	$Q (cd $(TOOLS_MODULE_DIR) && $(GO) mod tidy)
-
-.PHONY: diff
-diff:
```

Comment:
Is this part of the `goyek` workflow? If not can we keep it?

Reply:
https://github.com/signalfx/go-pipeline/blob/427de9c7dae208613be08ba02995b97b68a75b71/pipeline.go#L257-L282